core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/update.kt

```diff
@@ -14,6 +15,47 @@ class UpdateTests {
         df.update { col }.with { 2 } shouldBe df
     }
 
+    @DataSchema
+    interface AAndB {
+        val a: Int
+        val b: String
+    }
+
+    @DataSchema
+    data class Update(
+        override val a: Int,
+        override val b: String,
+        val c: Boolean,
+    ) : AAndB
+
+    @Test
+    fun `update asFrame`() {
+        val df = listOf(
+            Update(1, "a", true),
+            Update(2, "b", false),
```

Comment:
Sorry, probably I miss the idea, but where are the null data mentioned in the issue?

Reply:
That's irrelevant. The issue is that .where {} is not taken into account no matter what's inside it or which data is used